packages/headless/src/features/commerce/recommendations/recommendations-state.ts

```diff
@@ -10,11 +10,18 @@ export interface RecommendationsState {
   products: Product[];
 }
 
-export const getRecommendationsInitialState = (): RecommendationsState => ({
-  slotId: '',
-  headline: '',
-  error: null,
-  isLoading: false,
-  responseId: '',
-  products: [],
-});
+/**
+ * An object in which each key is a slot identifier, and each value is the corresponding recommendations slice.
+ */
+export type RecommendationsState = Record<string, RecommendationsSlice>;
+
+export const getRecommendationsInitialState = (): RecommendationsState => ({});
+export const getRecommendationsSliceInitialState =
+  (): RecommendationsSlice => ({
+    slotId: '',
```

Comment:
I'm not sure we need to keep the `slotId` here **and** as state key. I'll see if I can remove it.

Reply:
This was used in the request for recommendations. Now you pass that in the payload, removing its need to be stored in state.

Reply:
Fixed in https://github.com/coveo/ui-kit/pull/3801/commits/5b91a7e2afee7387758c1af891fcbed60c2a4a05